README.md

```diff
@@ -136,6 +135,10 @@ A curated list of resources for learning about natural language processing, text
 
 - [Entity Extraction and Network Analysis](http://brandonrose.org/ner2sna). Python, `StanfordCoreNLP`
 
+### Information Extraction
```

Comment:
Rather than introduce a new section, how about changing the title of the previous section to "Entity and Information Extraction" and putting this link there?

Reply:
Yes, that sounds reasonable.